scripts/globals/missions.lua

```diff
@@ -135,30 +135,30 @@ xi.mission.id =
     },
 
     -----------------------------------
-    --  Windurst (2)
+    --  Windurst - Interaction Framework (2)
     -----------------------------------
     [xi.mission.area[xi.mission.log_id.WINDURST]] =
     {
-        THE_HORUTOTO_RUINS_EXPERIMENT = 0,  -- ±
-        THE_HEART_OF_THE_MATTER       = 1,  -- ±
-        THE_PRICE_OF_PEACE            = 2,  -- ±
-        LOST_FOR_WORDS                = 3,  -- ±
-        A_TESTING_TIME                = 4,  -- ±
-        THE_THREE_KINGDOMS            = 5,  -- ±
-        THE_THREE_KINGDOMS_SANDORIA   = 6,  -- ±
-        THE_THREE_KINGDOMS_BASTOK     = 7,  -- ±
-        THE_THREE_KINGDOMS_SANDORIA2  = 8,  -- ±
-        THE_THREE_KINGDOMS_BASTOK2    = 9,  -- ±
-        TO_EACH_HIS_OWN_RIGHT         = 10, -- ±
-        WRITTEN_IN_THE_STARS          = 11, -- ±
-        A_NEW_JOURNEY                 = 12, -- ±
-        MAGICITE                      = 13, -- ±
-        THE_FINAL_SEAL                = 14, -- ±
-        THE_SHADOW_AWAITS             = 15, -- ±
-        FULL_MOON_FOUNTAIN            = 16, -- ±
-        SAINTLY_INVITATION            = 17, -- ±
-        THE_SIXTH_MINISTRY            = 18, -- ±
-        AWAKENING_OF_THE_GODS         = 19, -- ±
+        THE_HORUTOTO_RUINS_EXPERIMENT = 0,
+        THE_HEART_OF_THE_MATTER       = 1,
+        THE_PRICE_OF_PEACE            = 2,
+        LOST_FOR_WORDS                = 3,
+        A_TESTING_TIME                = 4,
+        THE_THREE_KINGDOMS            = 5,
+        THE_THREE_KINGDOMS_SANDORIA   = 6,
+        THE_THREE_KINGDOMS_BASTOK     = 7,
+        THE_THREE_KINGDOMS_SANDORIA2  = 8,
+        THE_THREE_KINGDOMS_BASTOK2    = 9,
+        TO_EACH_HIS_OWN_RIGHT         = 10,
+        WRITTEN_IN_THE_STARS          = 11,
+        A_NEW_JOURNEY                 = 12,
+        MAGICITE                      = 13,
+        THE_FINAL_SEAL                = 14,
+        THE_SHADOW_AWAITS             = 15,
+        FULL_MOON_FOUNTAIN            = 16,
+        SAINTLY_INVITATION            = 17,
+        THE_SIXTH_MINISTRY            = 18,
+        AWAKENING_OF_THE_GODS         = 19,
```

Comment:
These were re-written though?

Reply:
Yep, removed the individual comments and replaced in the header for that table